rest_framework-stubs/relations.pyi

```diff
@@ -31,7 +31,7 @@ _MT = TypeVar("_MT", bound=Model)
 _DT = TypeVar("_DT")  # Data Type
 _PT = TypeVar("_PT")  # Primitive Type
 
-class RelatedField(Field[_MT, _DT, _PT, Any]):
+class RelatedField(Field[_MT | PKOnlyObject, _DT, _PT, Any]):
```

Comment:
```suggestion
class RelatedField(Field[_MT, _DT, _PT, Any]):
```

This does not seem correct to me.
CC @UnknownPlatypus

Reply:
Sorry I have very little experience with `djangorestframework-stubs`, I'm not sure I'll be able to really help here